src/js/ui/editor/menu.js

```diff
@@ -80,6 +80,24 @@ export function renderSelectColorPosition(ctrl) {
         )
       ])
     ]),
+    m('div.select_input', [
+      m('label', {
+        'for': 'select_editor_positions'
+      }, 'Endgames'),
+      m('select.positions', {
+        id: 'select_editor_positions',
+        onchange: function(e) {
+          ctrl.loadNewFen(e.target.value);
+        }
+      }, [
+        optgroup('Set the board', [
```

Comment:
I said 'nothing happens' but I should say that I get this error in the console
'Uncaught SecurityError: Failed to construct 'Worker': Script at 'file:///home/sepiropht/src/lichobile/www/lib/socketWorker.js' cannot be accessed from origin 'null'.'